locust/test/test_runners.py

```diff
@@ -2224,11 +2230,15 @@ def my_task(self):
         with mock.patch("locust.rpc.rpc.Server", mocked_rpc()) as server:
             master = self.get_runner(user_classes=[TestUser])
 
-            run_count = [0]
+            run_count = [0, 0]
```

Comment:
Using a list here is more or less a hack for modifying values outside the function. I think this can be done in a more obvious way. even using a (properly named) global is preferable.